src/components/PageSubHeading.css

```diff
@@ -33,8 +33,7 @@
 }
 
 .page-sub-heading-description {
-  color: #777777;
-  font-style: italic;
+  color: #000000;
```

Comment:
Instead of making changes to this class (same feedback in PageHeading.css, I would update remove the use of this class from the component, and then apply whatever style you need to it through a style object. It could go into Theme as PageHeading and PageSubheading objects to make the transition a little more legible.